docs/configuration/private-registries/index.md

```diff
@@ -0,0 +1,33 @@
+# Private Registries
```

Comment:
Change this to Authenticated Registries, they don't need to be private

Reply:
Done